src/app/app.component.ts

```diff
@@ -26,4 +59,10 @@ export class AppComponent implements OnInit {
       }
     });
   }
+
+  ngOnDestroy(): void {
```

Comment:
Please delete type, it is not necessary for hooks

Reply:
fixed